ci/Jenkinsfile

```diff
@@ -1,33 +1,19 @@
 env.ghprbGhRepository = env.ghprbGhRepository ?: 'CentOS-PaaS-SIG/ci-pipeline'
 env.ghprbActualCommit = env.ghprbActualCommit ?: 'master'
-env.ghprbPullAuthorLogin = env.ghprbPullAuthorLogin ?: ''
-
-env.TARGET_BRANCH = env.TARGET_BRANCH ?: 'master'
 
 // Needed for podTemplate()
 env.CVENGINE_TAG = env.CVENGINE_TAG ?: 'stable'
 env.SLAVE_TAG = env.SLAVE_TAG ?: 'stable'
 
-env.DOCKER_REPO_URL = env.DOCKER_REPO_URL ?: '172.30.1.1:5000'
+env.DOCKER_REPO_URL = env.DOCKER_REPO_URL ?: '172.30.254.79:5000'
 env.OPENSHIFT_NAMESPACE = env.OPENSHIFT_NAMESPACE ?: 'continuous-infra'
 env.OPENSHIFT_SERVICE_ACCOUNT = env.OPENSHIFT_SERVICE_ACCOUNT ?: 'jenkins'
 
-// Audit file for all messages sent.
-msgAuditFile = "messages/message-audit.json"
-
-// IRC properties
-IRC_NICK = "contra-bot"
-IRC_CHANNEL = "#contra-ci-cd"
-
-// Number of times to keep retrying to make sure message is ingested
-// by datagrepper
-fedmsgRetryCount = 120
-
 // Execution ID for this run of the pipeline
 executionID = UUID.randomUUID().toString()
 
 // Pod name to use
-podName = 'fedora-atomic-' + executionID + '-' + TARGET_BRANCH
+podName = 'cvengine-' + executionID
 
 library identifier: "ci-pipeline@${env.ghprbActualCommit}",
```

Comment:
I'm not sure you want the $commit version of the ci-pipeline repo, you can probably just use the master version of the repo for the libraries.

Reply:
Here is an example of how I called the ci-pipeline libraries in a separate Jenkinsfile I created
https://github.com/johnbieren/systemd/blob/master/OpenShift/Jenkinsfile#L34

Reply:
👍